website/functions.js

```diff
@@ -0,0 +1,32 @@
+const readAloud = function(rowNum) {
```

Comment:
The 3 functions here are currently defined using Javascript's function expressions. But since we are not really using any of the benefits of function expressions, they could be defined as old school function declarations instead. See also [When to use a function declaration vs. a function expression](https://www.freecodecamp.org/news/when-to-use-a-function-declarations-vs-a-function-expression-70f15152a0a0/).